### ISSUE TITLE
--verbose-rpc flag on truffle console has no effect

### ISSUE THREAD

**Issue by grantwest** [trufflesuite/truffle#3412]
- [x] I've asked for help in the [Truffle Gitter](http://gitter.im/Consensys/truffle) before filing this issue.

---------------------------

## Issue

`--verbose-rpc` flag of `truffle console` command has no effect

## Steps to Reproduce

```
truffle console --network testnet --verbose-rpc
truffle(testnet)> (await web3.eth.getBalance(accounts[0])).toString()
'49935967720960000'
truffle(testnet)>
```

## Expected Behavior

I expect to see an RPC call logged.

## Actual Results

The only thing printed is the balance as shown in the Steps to Reproduce section above.

## Environment

* Operating System: Debian
* Truffle version: v5.1.42
* node version: v12.18.3
* npm version: 6.14.6

**Comment by eggplantzzz** [comment]
Very strange, when I connect to an existing `truffle develop` session I seem to get the rpc calls printed but not when I connect to Ropsten. We'll have to investigate, thank you for reporting this!

**Comment by shunjizhan** [comment]
same for me when using a custom local network, `--verbose-rpc` prints nothing extra

**Comment by sukanyaparashar** [comment]
This should be fixed in [v5.6.2](https://github.com/trufflesuite/truffle/releases/tag/v5.6.2)!
